app/controllers/articles_controller.rb

```diff
@@ -62,7 +62,7 @@ def create
   end
 
   def destroy
-    @article = Article.find(params[:id])
+    @article = Article.find_by(id: params[:id], user: current_user)
     if @article.destroy
```

Comment:
Similar here: we need to handle the case where `@article` is `nil` when there is no article that matches the search criteria:

```suggestion
    if @article&.destroy
```